scripts/gh-review-threads.sh

```
@@ -0,0 +1,276 @@
+done
+
+[[ -n "${SUBCOMMAND:-}" ]] || { print_usage; exit 1; }
+[[ -n "$OWNER" ]] || abort "--owner or GH_OWNER required"
+[[ -n "$REPO"  ]] || abort "--repo or GH_REPO required"
+[[ -n "$PR_NUMBER" ]] || abort "--pr or GH_PR required"
+
+info() { printf '%s\n' "$*" >&2; }
+
+# Fetch all review threads for the PR with pagination; outputs a JSON array of nodes
+fetch_threads() {
+  local after
```

Shellcheck (0.10.0)

[warning] after appears unused. Verify use (or export if used externally).

(SC2034)